packages/xod-cli/src/utils.js

```diff
@@ -16,4 +17,12 @@ export const runCommand = R.uncurryN(2, options =>
   )
 );
 
+// :: Nullable Path -> Path
+export const getWorkspacePath = R.compose(
+  expandHomeDir,
+  R.defaultTo(
+    process.env.XOD_WORKSPACE || DEFAULT_WORKSPACE_PATH
```

Comment:
I found that:

```js
export const DEFAULT_WORKSPACE_PATH = '~/xod/';
```

From my experience keeping trailing slashes while defining dirs lead to huge spaghetti in the future. As a rule of thumb: never end a path component with a slash, let the consumers add one if they want.

Can we fix it?

Reply:
Done